corehq/apps/enterprise/templates/enterprise/partials/project_tile.html

```diff
@@ -1,34 +1,53 @@
 {% load i18n %}
 
 <div
-  class="card text-center js-report-panel h-100" id="{{ report.slug }}"
+  class="card text-center js-report-panel h-100"
+  id="{{ report.slug }}"
   data-slug="{{ report.slug }}"
 >
   <div class="card-header">
     <h4 class="fs-5 pt-2">{{ report.title }}</h4>
   </div>
   <div class="card-body pb-1">
-    <div class="js-total fw-bold fs-1">
-      <i class="fa fa-spin fa-spinner"></i>
+    <div class="row justify-content-center">
+      <div class="col-auto align-items-center position-relative">
+        <div class="js-total fs-1">
+          <i class="fa fa-spin fa-spinner"></i>
+        </div>
+        {% if report.total_description %}
+          <div class="position-absolute start-100" style="top: 25%">
+            <span
+              class="hq-help-template"
+              data-content="{{ report.total_description }}"
+              data-placement="right"
+            >
+              <i class="fa fa-info-circle"></i>
```

Comment:
A note to myself: The span is already use the `hq-help-template`, the `fa-info-circle` is not required.